src/lib/useKeyfileGenerator.js

```diff
@@ -71,6 +74,8 @@ export default function useKeyfileGenerator(manualNetworkSeed) {
 
     const pair = deriveNetworkKeys(_networkSeed);
 
+    generateCode(pair.crypt.private + pair.auth.private + '42').then(setCode);
```

Comment:
The `'42'` here is probably some Urbit arcana? At the very least deserves a comment, if not a named `CONSTANT`.  
Might be better to move the entire concatenation here into `lib/networkCode` though, so that callers don't need to know anything about non-Bridge crypto.

Reply:
`42` here is actually `0x42`, ie `'B'`. This line is constructing a `+ring` for cryptosuite B. Do we know already have code to do this?